navigation-timing/nested_unload_timing.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="timeout" content="long">
+    <title>Navigation Timing: unload event with nested contexts</title>
+    <link rel="help" href="https://w3c.github.io/navigation-timing/"/>
+    <script src="/common/utils.js"></script>
+    <script src="/common/dispatcher/dispatcher.js"></script>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+
+    const dummyURL = () => `/navigation-timing/resources/blank_page_green.html?uid=${token()}`;
+
+    promise_test(async t => {
+        const mainWindowUnloadDuration = 100;
+        const iframeUnloadDuration = 400;
+        const popupUid = token();
+        const iframeUid = token();
+        const finalUid = token();
+        const popup = window.open(`resources/exec.html?uuid=${popupUid}`);
+        t.add_cleanup(() => popup.close());
+        const popupContext = new RemoteContext(popupUid);
```

Comment:
A few comments would be helpful here to understand the flow. e.g. I'm not sure what's `finalContext` and why it's needed..

Reply:
Added a long comment to describe the flow